frontend/src/pages/participatedMentoring/ReviewButton/ReviewButton.tsx

```diff
@@ -34,14 +35,23 @@ function ReviewCompleteButton({ onClick }: ReviewCompleteButtonProps) {
   );
 }
 
-function ReviewButton({ isReviewed, status }: ReviewButtonProps) {
+function ReviewButton({
+  isReviewed,
+  status,
+  onReviewButtonClick,
+}: ReviewButtonProps) {
   const canWriteReview = !isReviewed && status === StatusTypeEnum.COMPLETE;
 
   if (isReviewed) {
     return <ReviewCompleteButton onClick={() => {}} />;
   }
 
-  return <ReviewWriteButton onClick={() => {}} disabled={!canWriteReview} />;
+  return (
+    <ReviewWriteButton
+      onClick={() => onReviewButtonClick()}
```

Comment:
p3: 인자를 넘기지 않아서 onReviewButtonClick이 되어도 될 것 같아 👍
```suggestion
      onClick={onReviewButtonClick}
```

Reply:
[feat: 불필요한 인라인 함수 제거](https://github.com/woowacourse-teams/2025-Fit-toring/pull/417/commits/ab23e861c9bc0c1d08ad75bbac357bf954338ed4)